buildScripts/build.ps1

```diff
@@ -65,34 +79,52 @@ Try
     Write-Host "Running nuget restore for $solutionName"
     & $nugetPath restore "$BaseRepositoryDirectory\$solutionName.sln"
 
-    # Build
-    
-    Write-Host "Building $solutionName for EnvPlatform: $BuildPlatform Platform: $platform Configuration: $configuration"
-    $msbuildArgs = 
-    @(
-        ("$BaseRepositoryDirectory\$solutionName.sln"),
-        ("/p:Platform=$Platform"),
-        ("/p:Configuration=$Configuration"),
-        ("/restore"),
-        ("/binaryLogger:$BaseRepositoryDirectory\_build\$platform\$configuration\$solutionName.$platform.$configuration.binlog")
-    )
-
-    & $msbuildPath $msbuildArgs
-    if ($LASTEXITCODE -ne 0)
-    {
-        Write-Error "MSBuild failed with exit code $LASTEXITCODE"
-        exit $LASTEXITCODE
-    }
-
-    # Now create the nuget package 
+    # Create nuget pack properties that will always exist
     $nuspecFile = "$BaseRepositoryDirectory\src\ToolingNuget\nuget\Microsoft.Windows.VbsEnclave.CodeGenerator.nuspec"
     $nugetPackProperties = "target_version=$BuildTargetVersion;"
```

Comment:
>$nugetPackProperties [](http://example.com/codeflow?start=4&length=20)

pedant: fyi, usually nicer is to do use an array

    $nugetPackProperties = @();
    $nugetPackProperties += "...";
    $nugetPackProperties += "...";
    $nugetPackProperties += "...";
    $nugetPackPropertiesStr = $nugetPackProperties -Join ";";